package.json

```diff
@@ -29,6 +30,7 @@
     "react-native-svg-transformer": "^0.14.3",
     "react-native-vector-icons": "^8.1.0",
     "react-redux": "^7.2.4",
+    "react-thunk": "^1.0.0",
```

Comment:
Its better to delete one of the redux-thunk in the json by uninstall it then reinstall it again